packages/client/src/raffles/__fixtures__/getRaffle.textures.ts

```diff
@@ -0,0 +1,17 @@
+import { rest, RestHandler } from 'msw';
+import type { Raffle } from '../types';
+
+const path = '/api/checkout/v1/raffles/:raffleId';
+
+const fixtures = {
+  success: (response: Raffle): RestHandler =>
+    rest.get(path, async (_req, res, ctx) =>
```

Comment:
done

Reply:
Sorry for hijacking the comment :)
Regarding these unnecessary `async` in functions, we should have a rule in eslint to give an error in these cases. I tried to apply a plugin that had many rules besides the `require-await` one and a lot more errors were caught and then I dropped it but I can try applying only that specific rule so that these errors can be caught a lot sooner.